music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :genre, :author, :source, :label, :archived
+  attr_reader :id, :publish_date, :on_spotify
+
+  def initialize(id, publish_date, on_spotify)
+    super(publish_date, id)
+    @on_spotify = on_spotify
+  end
+
+  def can_be_archived?
+    super && @on_spotify
```

Comment:
- [ ] According to the project requirements, **can_be_archived?()** should return true if the parent's method returns true AND if on_spotify equals true. Please fix it in order to make your app works as expected 👍 

```ruby
   def can_be_archived?
      super && @on_spotify == true
    end
```

Reply:
hello @codecaiine it is the same, We are using short circuit and is the same as you said